Jellyfin.Api/Controllers/QuickConnectController.cs

```diff
@@ -66,6 +67,16 @@ public async Task<ActionResult<QuickConnectResult>> InitiateQuickConnect()
             }
         }
 
+        /// <summary>
+        /// Old version of <see cref="InitiateQuickConnect" /> using a GET method.
+        /// Still available to avoid breaking compatibility.
+        /// </summary>
+        /// <returns>The result of <see cref="InitiateQuickConnect" />.</returns>
+        [Obsolete("Use POST request instead")]
+        [HttpGet("Initiate")]
+        [ApiExplorerSettings(IgnoreApi = true)]
```

Comment:
Won't removing this from the api spec be a breaking change for the generated sdks?

Reply:
No because it uses the same name for the name method, so when using an old SDK version it will use the GET request and with newer SDK the POST request.